modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -144,6 +146,98 @@ public ObjectField addCustomObjectField(
 			state, false, objectFieldSettings);
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public ObjectField addObjectRelationshipObjectField(
+			String externalReferenceCode, User user,
+			Map<Locale, String> labelMap, String name,
+			ObjectDefinition objectDefinition1,
+			ObjectDefinition objectDefinition2, String readOnly,
+			String readOnlyConditionExpression, String relationshipType,
+			boolean required, boolean system)
+		throws PortalException {
```

Comment:
@devpedrotavares I'm not sure about the order here, but I would put the `ObjectDefinition objectDefinition1` and  `ObjectDefinition objectDefinition2` before `Map<Locale, String> labelMap` since `objectDefinitionId` comes first in the `service.xml`